### PR TITLE
update getting aggregator

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: pages/docs/getting-started/try-plasma-chamber-in-local.md [hunk 1]
```diff
@@ -17,11 +17,12 @@ For this quick start document using Typescript SDK, you need Node.js with the ve
...
 
-Get [plasma-aggregator](https://github.com/cryptoeconomicslab/wakkanay-plasma-aggregator).
+Get Plasma aggregator from [wakkanay repository](https://github.com/cryptoeconomicslab/wakkanay).
 
 ```bash
-$ git clone git@github.com:cryptoeconomicslab/wakkanay-plasma-aggregator.git
-$ cd wakkanay-plasma-aggregator
+$ git clone https://github.com/cryptoeconomicslab/wakkanay.git
+$ cd wakkanay
+$ cp -p integration-test/aggregator/.sample.env integration-test/aggregator/.env
```
Comment (daiki44) [inline pages/docs/getting-started/try-plasma-chamber-in-local.md:25]: @syuhei176 What do you think about we get the aggregator from `integration-test` directory?
I feel that is strange...

Comment (syuhei176) [reply pages/docs/getting-started/try-plasma-chamber-in-local.md:25]: I think so, talk about it later.

Comment (syuhei176) [reply pages/docs/getting-started/try-plasma-chamber-in-local.md]: fixed!ƪʃ

**Comment by daiki44** [comment]
@syuhei176 Will this change?
```
$ cp -p <your wakkanay-plasma-aggregator path>/config.local.json ./config.local.json
```

**Pull request merged**